BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc

```diff
@@ -46,19 +51,25 @@ void PATLowPtElectronSeedingEmbedder::produce(edm::StreamID, edm::Event &evt, ed
   evt.getByToken(ptBiased_src_, ptBiased);
   edm::Handle<edm::ValueMap<float> > unBiased;  
   evt.getByToken(unBiased_src_, unBiased);
+  edm::Handle<edm::ValueMap<float> > mvaId;  
+  evt.getByToken(mvaId_src_, mvaId);
 
 
   std::unique_ptr<pat::ElectronCollection> out(new pat::ElectronCollection);
 
-  for(auto ele : *lowpt) {
+  for ( size_t iele = 0; iele < lowpt->size(); ++iele ) {
+    edm::Ptr<pat::Electron> ptr(lowpt,iele);
```

Comment:
Why Ptr and not Ref?

Reply:
There's no particular reason here, but the choice was biased by my ntuplizer code, which requires a edm::Ptr to correctly deal with both reco::GsfElectrons and pat::Electrons, via an edm::View in the handle (to either collection) ...
I can change to a Ref if you prefer? Do we gain anything?

Reply:
I don't think so, just out of curiosity. We can keep it like this